alexthompson207/NY-Times--take-home-test
# Feature/all articles view

# NY Times -- Take Home PR 

### What's this PR do?

- Basic React App set -up 
- Create App, NewsView, and Article components with respective style files
- Create an API calls and utilities files


### Where should the reviewer start?

In the `App`, `NewView`, and `Article` component files.

### How should this be manually tested?

Clone/fetch this branch down, run `npm start` to host app locally. All articles from NYT top articles endpoint should appear and have basic styling.

### Any background context you want to provide?

N/A

### What are the relevant tickets?

#6 #1 

### Screenshots (if appropriate):

### Questions/next iterations:
- What information should `article details` include?
- Error handling
